SECURITY.md

```diff
@@ -2,12 +2,62 @@
+
+### CodeQL
+
+The [cFS CodeQL GitHub Actions workflow](https://github.com/nasa/cFS/actions/workflows/codeql-build.yml) is available to the public, however, the results are not available to the public. 
+
+CodeQL is ran for every push and pull-request on all branches of cFS in GitHub Actions. 
+
+For the CodeQL GitHub Actions setup, visit https://github.com/github/codeql-action. 
+
+### Cppcheck
+
+The [cFS Cppcheck GitHub Actions workflow and results](https://github.com/nasa/cFS/actions/workflows/static-analysis.yml) are available to the public. To view the results, select a workflow and download the artifacts. 
+
+Cppcheck is ran for every push on the main branch and every pull request on all branches of cFS in Github Actions. 
+
+For more information about Cppcheck, visit http://cppcheck.sourceforge.net/.
+
+### CodeSonar
+
+CodeSonar results are not available to the public. 
+
+CodeSonar is typically ran on a “once per release” sort of schedule.  
+
+For more information about CodeSonar, visit https://www.grammatech.com/codesonar-cc. 
+
+### LGTM
+
+LGTM [results](https://lgtm.com/projects/g/nasa/cFS?mode=list) are available to the public. Since suppression comments are not used, many warnings and errors may not be accurate. Therefore, the results may contain false positives and other inapplicable warnings and errors. 
+
+LGTM is ran for every pull-request on all branches of cFS.
+
+For more information about LGTM, visit https://lgtm.com/.
+
+### Fuzz Testing
+
+These [results](https://aetd-git.gsfc.nasa.gov/fsseb-582/fuzzing-cfs/-/pipelines) are available to the public for any logged in user. 
+
+The latest version of cFS is fuzzed nightly using a continuous integration pipeline. 
+
+For more information about Fuzzing cFS, visit https://aetd-git.gsfc.nasa.gov/fsseb-582/fuzzing-cfs. 
+
+## Security Reports
+
+To submit security reports or findings, email us at cfs-program@lists.nasa.gov.
 
 ## Additional Support
 
-For additional support, email us at cfs-program@lists.nasa.gov. For help using OSAL and cFS, [subscribe to our mailing list](https://lists.nasa.gov/mailman/listinfo/cfs-community) that includes all the community members/users of the NASA core Flight Software (cFS) product line. The mailing list is used to communicate any information related to the cFS product such as current releases, bug findings and fixes, enhancement requests, community meeting notifications, sending out meeting minutes, etc.
+For additional support, submit a GitHub issue or email us at cfs-program@lists.nasa.gov. [Subscribe to our mailing list](https://lists.nasa.gov/mailman/listinfo/cfs-community) that includes all the community members/users of the NASA core Flight Software (cFS) product line. The mailing list is used to communicate any information related to the cFS product such as current releases, bug findings and fixes, enhancement requests, community meeting notifications, sending out meeting minutes, etc.
```

Comment:
I'd say something like You can also email the cfs community at cfs-community@lists.nasa.gov; you can subscribe to the mailing list here (INSERT_URL)

Reply:
Updated to:

>For additional support, submit a GitHub issue. You can also email the cfs community at cfs-community@lists.nasa.gov. 
>
>You can subscribe to the mailing list [here](https://lists.nasa.gov/mailman/listinfo/cfs-community) that includes all the community members/users of the NASA core Flight Software (cFS) product line. The mailing list is used to communicate any information related to the cFS product such as current releases, bug findings and fixes, enhancement requests, community meeting notifications, sending out meeting minutes, etc.